src/app/Shared/Context/FilterContext.tsx

```diff
@@ -1,19 +1,20 @@
 import React, { useState, createContext } from "react";
-import { FilterInterface, mainFilterAttributes } from "../Model/Filter.model";
+import { FilterInterface } from "../Model/Filter.model";
 
-const generateEmptyFilter = (): FilterInterface => {
-    const defaultFilter = {} as FilterInterface;
-    mainFilterAttributes.forEach((element) => {
-        defaultFilter[element] = [];
-    });
-    return defaultFilter;
-};
-
-export const defaultFilter = generateEmptyFilter();
+interface FilterContextInterface {
+    mainFilter: string[];
+    selectedFilter: FilterInterface;
+    emptyFilter: FilterInterface;
```

Comment:
As i see in the first commit the emptyFilter is just an initial state for selectedFilter. If so please remove it and use a constant defaultValue. If the empyFilter depends on the API response set the selectedFilter to the default value accordingly to the API response.

Reply:
done. 
- Remvoce emptyFilter